docs/advanced/rules-object-expression.md

```diff
@@ -57,3 +57,29 @@ For rules with infinite parameters, you **must send** an array containing your s
 ```js
 const ruleObject = { one_of: [1, 2, 3, 4, 5, 6, 7, 8, 9] };
 ```
+
+## Conditionally Applying Rules
+
+There may be times where you need to conditionally apply rules depending on a condition in your application. This can easily be done using the JavaScript ternary operator.
+
+```js
+const enforceLengthCheck = true; // Some condition that determines whether the rule should be applied.
+const ruleObject = { required: true,  length: enforceLengthCheck ? { length: 10 } : false };
+```
+
+## Cross Field Validation
+It is also possible to use the **rule object expression** when you need to apply validation rules across fields.
+
+```js
+<ValidationObserver>
+  <ValidationProvider rules="{ required: true, confirmed: { target: '@confirmation' } }" v-slot="{ errors }">
```

Comment:
For brevity, you don't have to nest the confirmation field in an object:

```
"{ required: true, confirmed: '@confirmation' }
```

Reply:
@logaretm That's true for the `confirmed` rule. What I was trying to convey was the relationship of the parameters. I'll amend the example to use a custom rule with a few more parameters to better show what I mean.